lib/main.dart

```diff
@@ -11,103 +12,11 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'Flutter Demo',
```

Comment:
change app title to sen gp or something like that

Reply:
Done ✔️

![image](https://user-images.githubusercontent.com/46793415/132092619-78c0053a-d908-45aa-95bb-35ba2f57033c.png)